hikari/utilities/constants.py

```diff
@@ -81,4 +81,7 @@
 TWEMOJI_PNG_BASE_URL: typing.Final[str] = "https://github.com/twitter/twemoji/raw/master/assets/72x72/"
 TWEMOJI_SVG_BASE_URL: typing.Final[str] = "https://github.com/twitter/twemoji/raw/master/assets/svg/"
 
+# Miscellaneous
+DEFAULT_MAX_PRESENCES: typing.Final[int] = 25000
```

Comment:
I feel like we should ask Discord to provide this, because I feel uncomfortable with this potentially changing in the future without warning